### PR TITLE
add-contributing

### PR DESCRIPTION
**Pull request by zafarella**
so github will show link whenever PR are made

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is great you have something to contribute! 
+
+Before going any further please read the [wiki](https://github.com/iluwatar/java-design-patterns/wiki)
+with convemtions and rules we used for this project.
```
Comment (iluwatar) [inline CONTRIBUTING.MD]: Typo `convemtions`

**Comment by iluwatar** [comment]
@zafarella will gladly accept this. There is one small typo that should fixed beforehand.

**Comment by zafarella** [comment]
should be good

**Comment by iluwatar** [comment]
Thanks!

**Pull request merged**